src/rest/v0/transaction/mod.rs

```diff
@@ -0,0 +1,40 @@
+use crate::blockcfg::mock::Mockchain;
+use crate::intercom::TransactionMsg;
+use crate::utils::task::TaskMessageBox;
+use actix_web::error::ErrorBadRequest;
+use actix_web::{App, Error as ActixError, HttpMessage, HttpRequest, Responder};
+use bytes::IntoBuf;
+use chain_core::property::Deserialize;
+use chain_impl_mockchain::transaction::SignedTransaction;
+use futures::Future;
+use std::sync::{Arc, Mutex};
+
+pub type Task = Arc<Mutex<TaskMessageBox<TransactionMsg<Mockchain>>>>;
+
+pub fn crate_handler(
```

Comment:
Why is this named `crate_handler`?

Reply:
Because it creates item, which then can be passed to `add_handler`

Reply:
But what is a "crate", in this context?

Reply:
:D I'll fix it